app/views/publish_document/published.html.erb

```diff
@@ -1,4 +1,9 @@
-<% content_for :back_link, document_path(@document) %>
+<% if @document.review_state == "reviewed" %>
+  <% content_for :custom_back_link, {text: "Back to document list", href: documents_path}.to_json.html_safe %>
```

Comment:
Another idea to make this simpler: you could render the custom back link as a block right here instead of passing JSON hashes around.

Reply:
But wouldn't that render the component inside the `<main>` block?

Reply:
I don't think so. You can just yield the raw HTML in the `custom_back_link` section.